### ISSUE TITLE
What about repos whose default branch isn't called "master"?

### ISSUE THREAD

**Issue by callumlocke** [zeke/github-url-to-object#16]
It looks like this module always assumes the branch is called `'master'` if unspecified, but it would probably be safer to default to `""`. Not everyone uses the name "master" for their default branch.

**Comment by zeke** [comment]
Good call, @callumlocke. I'm not actively updating this repo these days, but will happily accept a PR that allows a custom branch name to be specified. Something like this:

```js
 gh(url, {defaultBranch: 'not-master'})
```
